eox_tenant/edxapp_wrapper/oauth_dispatch.py

```diff
@@ -0,0 +1,16 @@
+"""
+Oauth_dispatchs definitions.
+"""
+
+from importlib import import_module
+
+from django.conf import settings
+
+
+def get_edx_oauth2_validator_class():
+    """ Backend to get EdxOAuth2Validator class
```

Comment:
the double quotes must be in the same line if the text has only one line

Reply:
ty, solved